widget/playground/src/constants.ts

```diff
@@ -38,3 +40,123 @@ export enum SIDE_TABS_IDS {
 export const DEFAULT_PRIMARY_RADIUS = 20;
 export const DEFAULT_SECONDARY_RADIUS = 25;
 export const NOT_FOUND = -1;
+export const THEME: { id: Mode; title: string }[] = [
+  {
+    id: 'light',
+    title: 'Light',
+  },
+
+  {
+    id: 'dark',
+    title: 'Dark',
+  },
+
+  {
+    id: 'auto',
+    title: 'System',
+  },
+];
+
+export const CUSTOM_THEMES = [
+  {
+    id: 1,
+    dark: {
+      primary: '#1C3CF1',
+      secondary: '#2284ED',
+      neutral: '#222222',
+      background: '#010101',
+      foreground: '#FDFDFD',
+    },
+    light: {
+      primary: '#1C3CF1',
+      secondary: '#469BF5',
+      neutral: '#E6E6E6',
+      background: '#FDFDFD',
+      foreground: '#010101',
+    },
+  },
+  {
+    id: 2,
+    dark: {
+      primary: '#502f82ff',
+      secondary: '#9b6de2ff',
+      neutral: '#a4a0bd',
+      // info: '#000022',
+      foreground: '#fcfaffff',
+      background: '#120f29ff',
+    },
+    light: {
+      primary: '#31007aff',
+      secondary: '#653ba3ff',
+      neutral: '#84809d',
+      // info: '#8e6abf',
+      foreground: '#120f29ff',
+      background: '#fcfaffff',
+    },
+  },
+
+  {
+    id: 3,
+    dark: {
+      background: '#110114ff',
+      secondary: '#2d2a2dff',
+      primary: '#d400cbff',
+    },
+    light: {
+      background: '#fffeffff',
+      primary: '#d400cbff',
+      foreground: '#2f0146ff',
+      secondary: '#f5f1f7ff',
+      neutral: '#eae5eaff',
+    },
+  },
+  {
+    id: 4,
+    dark: {
+      primary: '#353038ff',
+      neutral: '#353038ff',
+      secondary: '#353038ff',
+      background: '#252028ff',
+      foreground: '#88818cff',
+    },
+    light: {},
```

Comment:
Do we need this line? Isn't light prop optional? (Also there are some similar cases)

Reply:
fixed